sqlglot/optimizer/pushdown_predicates.py

```diff
@@ -17,9 +21,13 @@ def pushdown_predicates(expression, dialect=None):
 
     Args:
         expression (sqlglot.Expression): expression to optimize
+        eliminate_join_marks (Optional[bool]): an override to eliminate legacy join marks
     Returns:
         sqlglot.Expression: optimized expression
     """
+    if eliminate_join_marks:
```

Comment:
why is this necessary? you can just call eliminate_jm before you call pushdown_predicates